### PR TITLE
docs: typo's & structure

### PR DESCRIPTION
**Pull request by ixje**
Was reading the documentation and noticed some typo's. The change on the `NeoVM stack items` was discussed on Discord

### CODE REVIEW

File: docs/neo-n3/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ __Maven__
 
 ### Devpack
 
-The neow3j devpack is a Java library that provides the Neo-specific functionality required to write smart contract. If
+The neow3j devpack is a Java library that provides the Neo-specific functionality required to write a smart contract. If
```
Comment (ixje) [inline docs/neo-n3/README.md:55]: alternative "write smart contract**s**"

File: docs/neo-n3/dapp_development/interacting_with_a_node.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ The stack included in the application logs will contain all the stack items that
 return stack is made up of one return value that is at index 0. You will need to know what type of stack item the
 invocation returns to be able to interpret it correctly.  
 
-Next to the return value you can also check what notification have been triggered by the transaction. The applications
+Next to the return value you can also check what notifications have been triggered by the transaction. The applications
```
Comment (ixje) [inline docs/neo-n3/dapp_development/interacting_with_a_node.md:122]: I was expecting this as "what notifications have" or "what notification has"

File: docs/neo-n3/dapp_development/interacting_with_a_node.md [hunk 3]
```diff
@@ -133,7 +133,7 @@ necessary methods to interact and make use of such wallets.
 First a wallet needs to be opened.
 
 ```java
-NeoOpenWallet response = neow.openWallet("/path/to/wallet.json", "walletPassword").send();
+NeoOpenWallet response = neow3j.openWallet("/path/to/wallet.json", "walletPassword").send();
```
Comment (ixje) [inline docs/neo-n3/dapp_development/interacting_with_a_node.md:136]: this is about using the same instance as was defined at the start in the document

File: docs/neo-n3/README.md (hunk 1 above)
Comment (mialbu) [reply docs/neo-n3/README.md:55]: I like the alternative better.

```suggestion
The neow3j devpack is a Java library that provides the Neo-specific functionality required to write smart contracts. If
```

**Review by mialbu — CHANGES_REQUESTED** [review]

**Review by mialbu — APPROVED** [review]
Great stuff, thanks for taking the time @ixje! 💯

**Pull request merged**